lib/recurly/paypal/braintree.js

```diff
@@ -89,24 +86,26 @@ export class BraintreePayPal extends Emitter {
       }
 
       // Tokenize with Recurly
-      this.recurly.request('post', '/paypal/token', { payload }, (error, token) => {
+      this.recurly.request('post', '/paypal/token', { type: 'braintree', payload }, (error, token) => {
```

Comment:
This is an API addition that lets new versions of the RJS API differentiate between braintree paypal tokenization and direct EC tokenization